src/modules/apis/getAppId.js

```diff
@@ -0,0 +1,21 @@
+const getAppId = async () => {
+  const response = fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/', {
+    method: 'POST',
+    // body: JSON.stringify(gameName),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const data = await (await response).text();
+  console.log(data); // eslint-disable-line
```

Comment:
- [ ] Please kindly remove commented codes and unnecessary `console.log`s from your code 👍